modules/oc-by-example-content.adoc

```diff
@@ -2209,18 +2240,6 @@ Run a proxy to the Kubernetes API server
 
 
 
-== oc registry info
```

Comment:
This command is being removed from this doc reference. I tested it, and when doing `oc registry --help`, `info` is listed in 4.14, but is not listed for 4.15. However, if I do `oc registry info --help`, it still does return with some values.

Do we know if this command is actually being removed in 4.15, or did something change that's not letting it appear if it should?

The help for it does say that it's experimental. But we have other experimental flags still show up in these docs (like `oc get-token` above in this PR).

Can anyone help confirm this? @ingvagabund @zhouying7780, or @stevsmit do you know anyone a registry SME who might know?

Reply:
`oc registry info` still exists and is not removed in 4.15

Reply:
The command has always been experimental, never promoted to stable. Though, for the sake of letting people know and have them ready let's do it.

> And maybe that's why it got removed from the CLI docs. I don't remember for sure, but we might not include commands that are deprecated. Do you know?

Unfortunately, I don't know.

Reply:
@soltysh Any chance you remember if our CLI doc generation script excludes deprecated CLI commands?

Reply:
@ingvagabund Do you know if there's a replacement for `oc registry info` that should be used instead? I see a few places in our docs where we use it.

* https://github.com/openshift/openshift-docs/blob/main/modules/disabling-redirect-private-storage-endpoint-azure.adoc?plain=1#L36
* https://github.com/openshift/openshift-docs/blob/main/modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc?plain=1#L65

Reply:
The closest replacement is to run `oc get imagestream -n openshift` and check the `IMAGE REPOSITORY` column.

Reply:
Opened https://issues.redhat.com/browse/OSDOCS-9586 to track updating references of "oc registry info" in the doc set to this alternative method.